### ISSUE TITLE
Some objects inactive after pull

### ISSUE THREAD

**Issue by nununo** [abapGit/abapGit#4513]
Hello,

I was told by more than one user that, when installing [abapBlame](https://github.com/abapinho/abapBlame), table type object `ZBLAME_PART_REF_T` is created but remains inactive. And indeed this is happening.

This table type's line is a reference to another class. I wonder if it remains inactive due to the order in which different object types are activated.

When I tried to test this in my own system, I upgraded `abapGit` to its latest version and, much to my surprise, two of it objects were also left in an inactive state.

Please let me know if you need any more information.

Thank you and regards,
Nuno

**Comment by larshp** [comment]
for ZBLAME_PART_REF_T suggest defining as a type in `zif_blame_object`, typically its not needed to have ddic referencing CLAS or interface objects

in abapGit, the ddic objects are activated first, before abap code

**Comment by larshp** [comment]
ref #606

**Comment by nununo** [comment]
Understood. Will do. Thanks!

**Comment by mbtools** [comment]
This should work

**Comment by mbtools** [comment]
Hi Nuno,

I tested with abapBlame on my 750 SP 19 system and the current developer version of abapGit (v1.105+).  I checked out commit 0d3387 which was the last before you removed `ZBLAME_PART_REF_T`.

All objects are pulled correctly without any errors. The table type is active and correctly referencing the class:

![image](https://user-images.githubusercontent.com/59966492/109215465-32b2d780-7781-11eb-9312-a0ff2bd27adf.png)

I'll close this again, but if you come across the problem again, feel free to open a new issue.


PS: "Checkout commit" was awesome for this type of analysis. Thx @mariusraht2 @EduardoCopat 👍
